site/_posts/2015-01-20-jekyll-meet-and-greet.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: news_item
+title: "Jekyll Meet & Greet at GitHub HQ"
+date: "2015-01-20 19:23:12 -0800"
+author: parkr
+categories: [meetup]
+---
+
+Hey! Our friends at GitHub have agreed to host a Jekyll meet & greet on
+**February 5, 2015 at 7pm**. The event will be hosted at [GitHub's
+Headquarters](https://www.google.com/maps/place/GitHub/@37.782084,-122.391403)
```

Comment:
Oh, and there is on last problem, I don't think I could book the flight, the Hummer and find the sticker I need to do my obligatory trolling when visiting San Francisco in time, but I do wish you guys the best with your meetup!